community/libcap/build

```diff
@@ -6,6 +6,7 @@ sed -i 's/pthread_yield/sched_yield/'                    libcap/psx.c
 
 make
 make \
+    GOLANG=no \
     PREFIX=/usr \
     SBINDIR=/usr/bin \
```

Comment:
Does this work when run from `make install`?

Reply:
wait don't merge this, you caught something here

Reply:
should be good now. tested it builds with and without 'go' installed on the system.